src/py/flwr/server/superlink/fleet/grpc_rere/server_interceptor_test.py

```diff
@@ -275,8 +276,14 @@ def test_successful_push_task_res_with_metadata(self) -> None:
         node_id = self.state.create_node(
             ping_interval=30, public_key=public_key_to_bytes(self._node_public_key)
         )
+        run_id = self.state.create_run("", "", "", {}, ConfigsRecord())
+        # Transition status to running. PushTaskRes is only allowed in running status.
+        _ = self.state.update_run_status(run_id, RunStatus(Status.STARTING, "", ""))
+        _ = self.state.update_run_status(run_id, RunStatus(Status.RUNNING, "", ""))
         request = PushTaskResRequest(
-            task_res_list=[TaskRes(task=Task(producer=Node(node_id=node_id)))]
+            task_res_list=[
+                TaskRes(task=Task(producer=Node(node_id=node_id)), run_id=run_id)
```

Comment:
I note we create a `TaskRes` in another test below this one. Should we pass the `run_id` in there too for consistency?

Reply:
Agreed, done.